projects/cds/README.md

```diff
@@ -0,0 +1,57 @@
+# Spartacus CDS configuration
+
+## CDS Module
+
+To enable the cds module in spartacus, add it to the list of imports in your root module, and create a CDSConfig using the Spartacus ConfigModule. The following is an example: 
+```
+@NgModule({
+  imports: [
+    CdsModule,
+    ConfigModule.withConfig(<CdsConfig>{
+      cds: {
+        tenant: 'example',
+        profileTag: {
+          javascriptUrl: 'http://tag.static.us.context.cloud.sap/js/profile-tag.js',
+          configUrl:
+            'https://tag.static.us.context.cloud.sap/config/example-123',
+        },
+      },
+    }),
+    ...
+```
+
+## Profiletag
+
+### CMS Component
+
+To enable profiletag, a CMSFlexComponent named 'ProfileTagComponent' will need to be created in the catalog. This component will then need to be placed into the footer. The following impex accomplishes this:
+```
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType;&componentRef;restrictions(uid,$contentCV)
+;;ProfileTagComponent;ProfileTag Spartacus Component;ProfileTagComponent;ProfileTagComponent;anonymousUserRestriction
```

Comment:
I see that you added `anonymousUserRestriction`. Is that because the tracking applies on to anonymous users?

Reply:
Ah, I may have misunderstoord this. I understood it to be that this would enable it for anonymous and logged in users. I guess in my case I should remove all restrictions?

Reply:
Yes, if you want it for both anonymous and registered user, just remove the restriction.